src/scripting/camera.cpp

```diff
@@ -100,12 +100,24 @@ Camera::set_scale(float scale)
   ease_scale(scale, 0, "");
 }
 
+void
+Camera::set_scale_target(float scale, float centerX, float centerY)
```

Comment:
Just wondering: Squirrel doesn't like default arguments, right?

Reply:
I tried. The issues I got were with how Miniswig defines wrapper functions. Not sure about Squirrel.

Reply:
right, *that* issue again.